app/assets/stylesheets/article-show.scss

```
@@ -556,6 +556,7 @@ article {
 .ff-container {
   --gif-bg: rgb(var(--black));
   --gif-color: rgb(var(--white));
+  --gif-btn-z-index: 2;
```

```suggestion
  // freezeframe uses a z-index of 1 to control when the static image is visible. 
  // To avoid clashes with our wider system of z-index values, we define a custom one for this use-case only
  --gif-btn-z-index: 2;
```

Let's add a comment in too, as otherwise we're gonna accidentally change it back to `--z-elevate` one day 😆